R/simulation-results.R

```diff
@@ -0,0 +1,7 @@
+
+#' @title SimulationResults
+#' @docType class
+#' @description  Results of a simulation run (either individual or population simulation)
+SimulationResults <- R6Class("SimulationResults",

```

Comment:
We have to think about what we need in this class. Just wrote it as an encapsulation for now